documentation/messages/user.registration.transactional.md

```diff
@@ -58,7 +58,7 @@ Accessed by connecting to RabbitMQ server and sending message in the following f
   subscribed: Boolean,
 
   /* Required. Unix timestamp. Example: `1458067430`. */
-  activity_timestamp: String,
+  activity_timestamp: Number,
```

Comment:
Would this be a good time to move to supporting Date objects with time zone details?